cmd/context/options.go

```diff
@@ -36,6 +36,7 @@ type ContextOptions struct {
 	IsOkteto              bool
 	raiseNotCtxError      bool
 	InsecureSkipTlsVerify bool
+	InferredToken         bool
```

Comment:
Even though I don't think this is the best way to do it, I think this is the only way to do it with the actual code so I'm approving it but leaving this comment for future re-reviews

Reply:
a refactoring would be necessary to avoid the approach followed. Currently, the context options are overwritten and it is difficult to know if the value is generated or given by the user